Modules/Office365DSC/DSCResources/MSFT_SPOSiteAuditSettings/MSFT_SPOSiteAuditSettings.psm1

```diff
@@ -0,0 +1,180 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param (
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $SiteUrl,
+
+        [Parameter(Mandatory = $true)]
+        [ValidateSet('All','None')]
+        [System.String]
+        $AuditFlags,
+
+        [Parameter()]
+        [ValidateSet("Present")]
+        [System.String]
+        $Ensure = "Present",
```

Comment:
Since Present is the only allowed option, wouldn't it be a better idea to skip this parameter and return Null as AuditFlags? Now Ensure=Absent does not mean the setting is absent, but that something has gone wrong.

Reply:
Good call. Now returning AuditFlags = $null if something goes wrong instead of Absent.

Reply:
Shouldn't the ensure parameter be removed altogether? Right now it is not being used at all.